tests/stdlib/tmitems.nim

```diff
@@ -134,3 +145,17 @@ block:
     x = <>Student(Name=x.attrs["Name"] & "foo")
   d[1].attrs["Name"] = "bar"
   echo d
+
+block:
+  var d = parseXml(newStringStream """<chapter>
+    <title>This is a Docbook title</title>
+    <para>
+        This is a Docbook paragraph containing <emphasis>emphasized</emphasis>,
+        <literal>literal</literal> and <replaceable>replaceable</replaceable>
+        text. Sometimes scrunched together like this:
+        <literal>literal</literal><replaceable>replaceable</replaceable>
+        and sometimes not:
+        <literal>literal</literal> <replaceable>replaceable</replaceable>
+    </para>
+</chapter>""",{reportComments, reportWhitespace})
+  echo d
```

Comment:
I am afraid `echo d` doesn't test if behavior is correct. 
We need something like this to test:
```nim

block:
  const para = """
        This is a Docbook paragraph containing <emphasis>emphasized</emphasis>,
        <literal>literal</literal> and <replaceable>replaceable</replaceable>
        text. Sometimes scrunched together like this:
        <literal>literal</literal><replaceable>replaceable</replaceable>
        and sometimes not:
        <literal>literal</literal> <replaceable>replaceable</replaceable>
"""
  var d = parseXml(newStringStream """<chapter>
    <title>This is a Docbook title</title>
    <para>""" & para &  """</para>
</chapter>""",{reportComments, reportWhitespace})

  doAssert(d.selectParaSomeHow == para)
```

Code above completely untested. Idea that we need `doAssert` checks to confirm spaces are preserved